File: C-Python_Solving_50_Problems/4-Hire_Decision_Program.py
# Step 1: Ask the user to enter his/her age
age = int(input("Step 1: Enter your age: "))

# Step 2: Ask the user if he/she has a driver license
hasDriverLicense = input("Step 2: Do you have a driver license? (Y/N): ")

# Step 3: Result = (Age > 21 and HasDriverLicense=True)
result = (age > 21 and (hasDriverLicense.upper() == 'Y'))

# Step 4: Check if result is True then Hire, otherwise Reject
# Step 5: Print "Hired" or "Rejected" accordingly
if result:
    print("Step 4 & 5: Hired")
else:
    print("Step 4 & 5: Rejected")

File: C-Python_Solving_50_Problems/4-Hire_Decision_Program.c
#include <stdio.h>

int main() {
    // Step 1: Ask the user to enter his/her age
    int age;
    printf("Step 1: Enter your age: ");
    scanf("%d", &age);

    // Step 2: Ask the user if he/she has a driver license
    char hasDriverLicense;
    printf("Step 2: Do you have a driver license? (Y/N): ");
    scanf(" %c", &hasDriverLicense);

    // Step 3: Result = (Age > 21 and HasDriverLicense=True)
    int result = (age > 21 && (hasDriverLicense == 'Y' || hasDriverLicense == 'y'));

    // Step 4: Check if result is True then Hire, otherwise Reject
    // Step 5: Print "Hired" or "Rejected" accordingly
    if (result) {
        printf("Step 4 & 5: Hired\n");
    } else {
        printf("Step 4 & 5: Rejected\n");
    }

    return 0;
}
